src/main/java/edu/kis/powp/jobs2d/command/CommandStatisticVisitor.java

```diff
@@ -0,0 +1,75 @@
+package edu.kis.powp.jobs2d.command;
+
+import java.util.Iterator;
+import java.util.Vector;
+
+public class CommandStatisticVisitor implements CommandVisitor {
+    private double totalLength = 0;
+    private Vector<Integer> previousCoordinates = null;
+    private Vector<Integer> currentCoordinates = null;
+    private int operateToCount = 0;
+
+    private int setPositionCount = 0;
+
+    public double getTotalLength() {
+        return totalLength;
+    }
```

Comment:
There should be at least two distances: the total distance of devices head and the distance of performed operateTo calls.